dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentOperationParamsMapperTest.java

```diff
@@ -147,7 +151,12 @@ void shouldMapWithoutFetchingNullParamsWhenParamsAreNotSpecified()
     EnrollmentOperationParams operationParams =
         EnrollmentOperationParams.builder().orgUnitMode(ACCESSIBLE).build();
 
-    mapper.map(operationParams);
+    try (MockedStatic<CurrentUserUtil> userUtilMockedStatic = mockStatic(CurrentUserUtil.class)) {
```

Comment:
Could you remove the static mocks and fix the injection of the user in the setup?
I think the user is injected before
`user.setTeiSearchOrganisationUnits(Set.of(orgUnit1, orgUnit2));
    user.setOrganisationUnits(Set.of(orgUnit2));`

And then you can probably not find the search and capture scope that you need